scipio/src/executor.rs

```diff
@@ -534,6 +534,7 @@ impl LocalExecutor {
     fn init(&mut self) -> io::Result<()> {
         let queues = self.queues.clone();
         let index = 0;
+        let queues_weak = Rc::downgrade(&queues);
```

Comment:
The clone above seems superfluous

Reply:
humm, that looks like the kind of stuff clippy would complain about. It usually complains a lot about clones that are not needed